config/env/development.js

```diff
@@ -22,7 +22,8 @@ module.exports = {
     }
   },
   app: {
-    title: defaultEnvConfig.app.title + ' - Development Environment'
+    title: defaultEnvConfig.app.title + ' - Development Environment',
+    domain: 'localhost:3000'
```

Comment:
Could this be changed to `domain: process.env.DOMAIN || 'localhost'`? and added to the test env config as well?